python/neuroglancer/pipeline/task_queue.py

```diff
@@ -183,21 +190,43 @@ def cloud_insertion(api):
         else:
             cloud_insertion(self._api)
 
+        return self
+
+    def wait(self):
+        return self.wait_until_queue_empty()
+
     def wait_until_queue_empty(self):
         self._queue.join()
+        return self
 
     def get(self):
         """
-        Gets the named task in a TaskQueue.
+        Gets information about the TaskQueue
         """
-        raise NotImplemented
+        return self._api.taskqueues().get(
+            project=self._project,
+            taskqueue=self._queue_name,
+            getStats=True,
+        ).execute(num_retries=6)
+
+    def get_task(self, tid):
```

Comment:
The reason why I was thinking we should be explicit about the ID is that list and get return useful information, but in different forms, but you couldn't call the stuff returned by list a task, just task metadata. So I figured you just supply what is needed. I'm not shy about throwing IDs around so long as they're immutable. Alternatively, we could format the return of list into some kind of task. 

By return one, you mean you'd like a return multiple? That would be good. I just stubbed this out because I was using `get` and wanted to draw a clear distinction.